packages/ComponentLibrary/src/components/Table/index.tsx

```diff
@@ -146,7 +195,7 @@ const Table: React.FC<TableProps> = ({ data, isTreeStructure = false }) => {
             ...tableStyles.tablePaper,
             width: isSidebarOpen ? 'calc(70% - 0.5rem)' : '100%',
           }}>
-          <MaterialReactTable table={table} />
+          {renderContent()}
```

Comment:
Please rename the "renderContent" function to something like "Content" and use it as a component:
<Content />